File: examples/after/sample.js
// Example JavaScript code with deprecated API
const oldApi = require('new-api');

function main() {
    const client = oldApi.createClient({ key: 'secret' });
    const user = newApi.getUserById(123);
    const posts = newApi.getUserPosts(123);
    return { user, posts };
}

main();

File: examples/after/sample.py
# Example Python code with deprecated API
from new_api import Client

def main():
    client = Client(api_key="secret")
    user = new_api.fetch_user_by_id(user_id=123)
    posts = new_api.list_posts_by_user(user_id=123)
    return user, posts

if __name__ == "__main__":
    main()
